main.cc

```diff
@@ -30,26 +30,26 @@ vector<Layer*> layers;
 vector<Sample> trainData, testData;
 
 Real lr = 0.001;
-Real momentum = 0.9;
+Real momentum = 0;
 Real weightDecay = 0.004;
 
 void getLayers(vector<Layer*> *layers) {
 
   layers->push_back(new InputLayer("input", batch, 32, 32, 3));
 
-  layers->push_back(new ConvolutionalLayer("conv1", 32, 5, 1, 2, layers->back(), &gaussianFiller1, &constantFiller));
-  layers->push_back(new MaxPoolingLayer("pool1", 3, 2, layers->back()));
-  layers->push_back(new ReluLayer("relu1", layers->back()));
+//  layers->push_back(new ConvolutionalLayer("conv1", 32, 3, 1, 1, layers->back(), &gaussianFiller1, &constantFiller));
```

Comment:
Would you mind create a new method to return your config? e.g. another `getLayers` method.